mars/tensor/expressions/datastore/totiledb.py

```diff
@@ -70,6 +72,18 @@ def _get_out_chunk(cls, op, in_chunk):
         return chunk_op.new_chunk([in_chunk], shape=out_chunk_shape,
                                   index=in_chunk.index)
 
+    @classmethod
+    def _process_out_chunks(cls, op, out_chunks):
+        if len(out_chunks) == 1:
+            return out_chunks
+
+        consolidate_op = TensorTileDBConsolidate(
```

Comment:
Should we introduce tree dependency ?

Reply:
No, I think tree dependency is just a tradeoff, and looks uncomfortable, I hope we could discuss sort of new dependency except for data dependency, until then, we could resolve such operand.

Reply:
Agree.